benchmark/single-source/RemoveWhere.swift

```diff
@@ -0,0 +1,205 @@
+//===--- RemoveWhere.swift ------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import TestsUtils
+
+public let RemoveWhere = [
+  // tests repeated remove(at:) calls in generic code
+  BenchmarkInfo(name: "AARemoveWhereQuadraticStrings", runFunction: run_RemoveWhereQuadraticStrings, tags: [.validation, .api], setUpFunction: buildWorkload),
```

Comment:
Oops. I forgot to remove the "AA"s. Will do that after the benchmark results come back.